nifi-extension-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestGenerateFlowFile.java

```diff
@@ -73,4 +78,30 @@ public void testDynamicPropertiesToAttributes() {
         generatedFlowFile.assertAttributeEquals("mime.type", "application/text");
     }
 
+    @Test
+    public void testExpressionLanguageSupport() {
+        TestRunner runner = TestRunners.newTestRunner(new GenerateFlowFile());
+        runner.setProperty(GenerateFlowFile.FILE_SIZE, "${random():mod(10):plus(1)}B");  // represents a range of 1-10 bytes, inclusive
+        runner.setProperty(GenerateFlowFile.UNIQUE_FLOWFILES, "true");
+        runner.assertValid();
+
+        // Execute multiple times to ensure adequate distribution of file sizes
+        runner.run(1000);
+
+        runner.assertTransferCount(GenerateFlowFile.SUCCESS, 1000);
+        List<MockFlowFile> flowFiles = runner.getFlowFilesForRelationship(GenerateFlowFile.SUCCESS);
+        Map<Long, Integer> fileSizeDistribution = new HashMap<>();
+        flowFiles.forEach(ff -> {
+            long fileSize = ff.getSize();
+            fileSizeDistribution.put(fileSize, fileSizeDistribution.getOrDefault(fileSize, 0) + 1);
+        });
+        long minSize = fileSizeDistribution.keySet().stream().min(Long::compareTo).orElse(-1L);
+        long maxSize = fileSizeDistribution.keySet().stream().max(Long::compareTo).orElse(-1L);
+
+        // Assert all file sizes fall within the range and that there exists more than one unique file size value
+        Assertions.assertTrue(minSize > 0 && minSize < maxSize);
+        Assertions.assertTrue(maxSize <= 10);
```

Comment:
The import static should be used for `assertTrue`.

Reply:
Done.